src/main/java/com/github/black0nion/blackonionbot/systems/antispoiler/AntiSpoilerSystem.java

```diff
@@ -18,14 +20,14 @@ public class AntiSpoilerSystem {
 	 * @return if the message contained a spoiler
 	 */
 	@SuppressWarnings("ConstantConditions")
-	public static boolean removeSpoilers(final SlashCommandEvent event) {
+	public static boolean removeSpoilers(final @NotNull SlashCommandEvent event, @NotNull SlashCommandInteractionEvent interactionEvent) {
 		final BlackGuild guild = event.getGuild();
-		final Message msg = event.getMessage();
-		final String message = msg.getContentRaw();
 		final BlackUser author = event.getUser();
+		final Message msg = interactionEvent.getTextChannel().retrieveMessageById(interactionEvent.getTextChannel().getLatestMessageIdLong()).complete();
```

Comment:
This probably won't work because we also want to remove spoilers on non-interaction events